src/app/components/input/input.component.html

```diff
@@ -7,9 +7,9 @@
 </div>
```

Comment:
same as above the name of the component should be more descriptive and meaningful instead of just the input